agent/engine/docker_task_engine.go

```diff
@@ -615,6 +615,14 @@ func (engine *DockerTaskEngine) startContainer(task *api.Task, container *api.Co
 	return client.StartContainer(dockerContainer.DockerID, startContainerTimeout)
 }
 
+func (engine *DockerTaskEngine) provisionContainerResources(task *api.Task, container *api.Container) DockerContainerMetadata {
+	seelog.Infof("Task [%v]: Setting up container resources for container [%v]", task.String(), container.String())
+
+	// TODO: Invoke libcni here
+
+	return DockerContainerMetadata{}
```

Comment:
This is just placeholder code. It doesn't break the functionality as it exists today. As per the comment, `libcni` invocation is supposed to happen here.